src/components/layout/footer/footer.tsx

```diff
@@ -1,43 +1,30 @@
-import { Box, Container, Flex, Grid, Link, Text } from 'theme-ui';
-import { Booking } from '../../booking';
-import { Logo } from '../../logo';
-import { NetworksLinks } from './networks-links';
+import { Box, Container, Flex, Link } from 'theme-ui';
 
 export const Footer: React.FC = () => (
-  <Box sx={{ minHeight: '5vh', background: 'black80', mt: 'auto' }}>
-    <Container variant="readable">
+  <Box sx={{ minHeight: '5vh', background: 'dark100', mt: 'auto' }}>
+    <Container variant="full">
       <Flex
-        sx={{ flexDirection: 'column', py: 5, gap: 5, color: 'background' }}
+        sx={{
+          color: 'background',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          fontStyle: 'normal',
+          fontWeight: 400,
+          lineHeight: '23px',
+          letterSpacing: -0.02,
+          backgroundSize: 'cover',
```

Comment:
This seems like a lot to handle in a single Flex component. Maybe consider moving it to theme?

Reply:
Agreed, but it was mostly superfluous code so shortened to:
          color: 'background',
          justifyContent: 'space-between',
          alignItems: 'center',